src/main/java/com/iexec/common/config/PublicConfiguration.java

```diff
@@ -27,12 +27,13 @@
 public class PublicConfiguration {
     String workerPoolAddress;
     String schedulerPublicAddress;
-    String blockchainAdapterUrl;
+    String configServerUrl;
```

Comment:
Wouldn't we prefer a proper deprecation notice here or at least `@JsonAlias` on the new field name ?